src/permission/fs_permission.h

```diff
@@ -126,6 +139,9 @@ class FSPermission final : public PermissionBase {
     void Insert(const std::string& s);
     bool Lookup(const std::string_view& s) { return Lookup(s, false); }
     bool Lookup(const std::string_view& s, bool when_empty_return);
+    std::string NormalizePathIfCaseInsensitive(const std::string& path);
```

Comment:
Nit:

```suggestion
    std::string NormalizePathIfCaseInsensitive(const std::string_view path);
```

Reply:
This will create an unnecessary copy. Considering I'm calling `std::tolower` which needs a `std::string` representation (or use an ineffective iteration over chars).